code/game/machinery/civilian_bounties.dm

```diff
@@ -130,7 +130,7 @@
 	pot_acc.bounties = crumbs
 
 /obj/machinery/computer/piratepad_control/civilian/proc/pick_bounty(choice)
-	if(inserted_scan_id?.registered_account)
+	if(!inserted_scan_id || !inserted_scan_id.registered_account)
```

Comment:
I don't understand, why didn't you just invert the existing null coalescence check?

Reply:
changed it and the machine worked again

Reply:
```suggestion
	if(!inserted_scan_id?.registered_account)
```